spec/exercise_fail_spec.rb

```diff
@@ -0,0 +1,80 @@
+          blueprint_id: blueprint[:id],
+          name: "Side Lateral Raise",
+          instructions:
+            "Pick a couple of dumbbells and stand with a straight torso and the dumbbells by your side at arms length with the palms of the hand facing you. This will be your starting position. While maintaining the torso in a stationary position (no swinging), lift the dumbbells to your side with a slight bend on the elbow and the hands slightly tilted forward as if pouring water in a glass. Continue to go up until you arms are parallel to the floor. Exhale as you execute this movement and pause for a second at the top. Lower the dumbbells back down slowly to the starting position as you inhale. Repeat for the recommended amount of repetitions. Variation: This exercise can also be performed sitting down.",
+          weight: 6,
+          reps: 9,
+          user_id: user[:id]
+        )
+      2.times do
+        ExerciseSet.create(
+          weight: 6,
+          reps: 9,
+          exercise_id: less_than_half_unsuccessful.id,
+          completed_at: DateTime.now,
+          user_id: user[:id]
+        )
+        ExerciseSet.create(
+          weight: 4,
+          reps: 7,
+          exercise_id: less_than_half_unsuccessful.id,
+          completed_at: DateTime.now,
+          user_id: user[:id]
+        )
+      end
+
+      previous_exercise_unsuccessful_half_or_more =
+        Exercise.create(
+          blueprint_id: blueprint[:id],
+          name: "Dumbbell side bend",
+          instructions:
+            "Stand up straight while holding a dumbbell on the left hand (palms facing the torso) as you have the right hand holding your waist. Your feet should be placed at shoulder width. This will be your starting position. While keeping your back straight and your head up, bend only at the waist to the right as far as possible. Breathe in as you bend to the side. Then hold for a second and come back up to the starting position as you exhale. Tip: Keep the rest of the body stationary. Now repeat the movement but bending to the left instead. Hold for a second and come back to the starting position. Repeat for the recommended amount of repetitions and then change hands. Caution: Refrain from using this exercise if your obliques tend to grow easily as wide obliques take away from your symmetry. Variations: You can also do this exercise while seating on a bench or with a barbell.",
+          weight: 5,
+          reps: 10,
+          user_id: user[:id]
+        )
+      4.times do
+        ExerciseSet.create(
+          weight: 4,
+          reps: 7,
+          exercise_id: previous_exercise_unsuccessful_half_or_more.id,
+          completed_at: DateTime.now,
+          user_id: user[:id]
+        )
+      end
+      Exercise.create_exercise(blueprint[:id], workout_id, sets, user[:id])
+    end
+
+    it "previous exercise unsuccessful(User less than or equal to half the previous set at the less than the assigned weight and reps)" do
```

Comment:
I'd expect this test to have a title like:
```ruby
 it "expects thingy to happen" do
```

Reply:
I updated all the test names. Please let me know if these are better or if they are too wordy